### DISCUSSION TITLE
9/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#118] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimento

**Comment by Pamela198713** [comment]
Ayer tuve clases en el ispc y consulte dudas del trabajo a la profe.
Hoy tengo otra materia veré de completar el trabajo con las chicas para la primer entrega.
Sigue siendo impedimento el tiempo y la organización para las tareas que tengo.

**Comment by rpgrca** [comment]
Ayer estuve en la clase de Tecnología y Desarrollo donde vimos un ejemplo del FODA. También empecé con la parte de registración y empecé a agregar código a la app para hacer la conexión con la api web. Espero poder terminarlo hoy y hacer algunas pruebas para ver si funciona correctamente. Hoy tengo clase de Práctica, esta vez voy a estar!
